components/server/src/bitbucket/bitbucket-file-provider.ts

```diff
@@ -15,10 +15,7 @@ export class BitbucketFileProvider implements FileProvider {
     @inject(BitbucketApiFactory) protected readonly apiFactory: BitbucketApiFactory;
 
     public async getGitpodFileContent(commit: Commit, user: User): Promise<MaybeContent> {
-        const yamlVersion1 = await Promise.all([
-            this.getFileContent(commit, user, ".gitpod.yml"),
-            this.getFileContent(commit, user, ".gitpod"),
```

Comment:
I don't know where all the API calls are coming from, but asking for `.gitpod` every time is unnecessary.

Reply:
Do we have any metrics that would allow us to trace this? Could we add the metrics first to understand the source of these requests?

Reply:
I agree we need to better understand why we run into the limit (atlassian is unclear about that). But we have customers sitting on a bad experience today and removing this call does make sense in anyway even if it doesn't help with the rate limit. So I'd like to go ahead with this change and follow up with investigation of the api calls.